smart_card_connector_app/src/window-main.js

```diff
@@ -58,7 +59,9 @@ GSC.ConnectorApp.Window.AppsDisplaying.initialize();
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
-GSC.PopupWindow.Client.setExplicitWindowTitle();
 GSC.PopupWindow.Client.showWindow();
 
+// This call does all of the translation work.
```

Comment:
This comment doesn't provide much useful information. I guess, the function name and its JSDoc already provide with all the necessary context.